addons/web/static/tests/views/view_dialogs/export_data_dialog.test.js

```diff
@@ -670,7 +670,12 @@ test("ExportDialog: export all records of the domain", async () => {
     onRpc("/web/export/formats", () => {
         return [{ tag: "xls", label: "Excel" }];
     });
-    onRpc("/web/export/get_fields", () => {
+    onRpc("/web/export/get_fields", async (request) =>  {
+        const { params } = await request.json();
+        if (isDomainSelected) {
+            const expectedDomain = params.parent_field ? [] : [["bar", "!=", "glou"]];
+            expect(params.domain).toEqual(expectedDomain, {message: "Domain is only applied on the root model"});
```

Comment:
add an expect.step here to ensure that this assertion is done (at least twice I guess)